common/build.gradle

```diff
@@ -170,6 +170,7 @@ dependencies {
     testImplementation 'org.json:json:20160810'
     testImplementation "junit:junit:$rootProject.ext.junitVersion"
     testImplementation "org.mockito:mockito-core:$rootProject.ext.mockitoCoreVersion"
+    testImplementation "org.mockito.kotlin:mockito-kotlin:3.6.28"
```

Comment:
Have we done a size increase analysis on this one? Relavent if we are adding it to common as it is consumed by OneAuth

Reply:
This is a test dependency, so this isn't going to ship

Reply:
Can you add the version number to versions.gradle file?